pkg/strings/strings.go

```diff
@@ -1,10 +1,9 @@
 package strings
 
+import "slices"
+
+// Deprecated: Use go slices package instead.
```

Comment:
Use Go [doc links](https://tip.golang.org/doc/comment#doclinks) instead:


```suggestion
// Deprecated: Use standard library [slices.Contains] instead.
```